File: Cell.java
public class Cell {
    private boolean hit;
    private boolean ship;
    private String id;
    
    public void setHit(boolean h){
        hit = h;
    }
    public void setShip(boolean s){
        ship = s;
    }
    public void setId(String s){
        id = s;
    }
    
    public boolean getHit(){
        return hit;
    }
    public boolean getShip(){
        return ship;
    }
    public String getId(){
        return id;
    }
}

File: cell.py
"""
Description: Class for cell object
<pre>
Name: Gabriel Krishnadasan, Alizea Hinz, Aidan Rooney, Marissa Nicole Esteban (Pascal)
Course: COMP-305 FA22
Professor: A. Nuzen
</pre>
"""
class cell(object):
    def __init__(self, h:bool, s:bool, i:str, sh:bool)->None:
        """
        Constructor for cell object
        """
        self.hit = h
        self.ship = s
        self.id = i
        self.successful_hit = sh

    def __repr__(self)->str:
        """
        String representation of a cell object
        """
        return str(self.hit) + " "+str(self.ship) + " "+self.id + " "+self.successful_hit

